src/strings/en-gb.json

```diff
@@ -1417,5 +1417,29 @@
     "ButtonUseQuickConnect": "Use Quick Connect",
     "ButtonActivate": "Activate",
     "Authorize": "Authorise",
-    "EnableAutoCast": "Set as Default"
+    "EnableAutoCast": "Set as Default",
```

Comment:
The key seems very different from its value here. It's usually better to make it somewhat similar, to help translators.